### PR TITLE
Color bar & option input components

### LINKED ISSUE #47: Add simple color input
screen shot 2016-02-04 at 15 11 51

### PR DESCRIPTION
**Pull request by MariaCheca**
Closes #47 

Color bar:
screen shot 2016-02-23 at 11 11 01

Option input:
screen shot 2016-02-23 at 11 11 14

CR @piensaenpixel cc @xavijam

### CODE REVIEW

File: src/scss/cdb-components/forms/color_input.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+/* SG
+# Forms/Color input
+
+```
+<fieldset>
+  <p class="CDB-Legend u-iBlock CDB-Text is-semibold CDB-Size-small u-rSpace--m">COLOR</p>
+  <div class="u-iBlock">
+    <div class="CDB-ColorInput">
+      <span class="CDB-ColorInput-colorBar Color-Higlight"></span>
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/color_input.scss]: For me this is a differente component, because we should reuse in some place

**Comment by piensaenpixel** [comment]
@MariaCheca could you update the screenshot? :dancer:

**Comment by piensaenpixel** [comment]
:+1:

**Comment by MariaCheca** [comment]
:sparkles:

**Pull request merged**